src/ProductTour/CheckpointHeader.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage, useIntl } from 'react-intl';
+
+import Icon from '../Icon';
+import IconButton from '../IconButton';
+import { Close } from '../../icons';
+import CheckpointTitle from './CheckpointTitle';
+import messages from './messages';
+
+const CheckpointHeader = React.forwardRef(({
+  index, onDismiss, title, totalCheckpoints,
+}) => {
+  const intl = useIntl();
+  const oneBasedIndex = index + 1;
+
+  const closeAltText = intl.formatMessage({
```

Comment:
Should these translations be in the [messages.js](https://github.com/openedx/paragon/blob/release-22.x/src/ProductTour/messages.js#L3) file?